Assets/PackageStagingArea/eu.netherlands3d.cameras/Runtime/Scripts/Cameras/OrthographicSwitcher.cs

```diff
@@ -0,0 +1,301 @@
+        private FreeCamera freeCamera;
+        private Transform cameraTransform;
+        private float previousPitchWhenSwitchingToAndFromOrtho = 60f;
+        private float previousFovWhenSwitchingToAndFromOrtho = 60f;
+        private Sequence animationSequence;
+
+        [Header("Animation tweaks")]
+        [SerializeField] private float repositioningDuration = 0.3f;
+        [Range(0f, 60f)] [SerializeField] private float simulatedOrthoFov = 2f;
+        [Tooltip("The animation dynamically heightens the Simulated Orthogonal FOV if the predicted height of the camera exceeds this number")]
+        [SerializeField] private float maxCameraHeightWhenSimulating = 8000f;
+
+        private void Awake()
+        {
+            cameraComponent = GetComponent<Camera>();
+            cameraTransform = cameraComponent.transform;
+            freeCamera = GetComponent<FreeCamera>();
+
+            // Initialize orthographicSize for calculations
+            cameraComponent.orthographicSize = transform.position.y;
+            
+            // Initialize cached value with a preset
+            previousPitchWhenSwitchingToAndFromOrtho = cameraTransform.eulerAngles.x;
+            previousFovWhenSwitchingToAndFromOrtho = cameraComponent.fieldOfView;
+        }
+
+        private void Update()
+        {
+            if (cameraComponent.orthographic)
+            {
+                // By dividing it by the field of view divided by 100, the size matches close to what you would see
+                // in perspective mode
+                cameraComponent.orthographicSize = transform.position.y * (previousFovWhenSwitchingToAndFromOrtho / 100f); 
+            }
+        }
+
+        /// <summary>
+        /// Switch camera to ortographic mode and limit its controls
+        /// </summary>
+        /// <param name="enableOrthographic">Ortographic mode enabled</param>
+        public void EnableOrthographic(bool enableOrthographic)
+        {
+            // Switch back to perspective
+            if (!enableOrthographic)
+            {
+                SwitchToPerspectiveMode();
+                return;
+            }
+
+            SwitchToOrthographicMode();
+        }
+
+        private void SwitchToOrthographicMode()
+        {
+            if (cameraComponent.orthographic) return;
+
+            // Remember the current pitch so that we have the correct angle to switch back to
+            // when re-entering perspective
+            previousPitchWhenSwitchingToAndFromOrtho = cameraTransform.eulerAngles.x;
+
+            // Pull forward camera to make sure that what is in center of the screen stays there
+            var cameraLookWorldPosition = freeCamera.GetWorldPoint(
+                new Vector3(Screen.width * 0.5f, Screen.height * 0.5f, 0)
+            );
+            cameraLookWorldPosition.y = transform.position.y;
+
+            // Look downwards
+            var flattenedForward = cameraTransform.forward;
+            flattenedForward.y = 0;
```

Comment:
niet zeker of dit uitmaakt, maar ik zou hierna nog `flattendForward.Normalize()` doen

Reply:
Aangezien het nu goed lijkt te werken zou dat alleen een extra operation zijn; ik laat dat even achterwege tot het nodig blijkt te zijn